cli/tests/integrations/helpers/common.py

```diff
@@ -204,9 +204,9 @@ def update_config(name, value, env=None):
     # we need to also restore the token if this occurs
     token = None
     if name == "core.dcos_url":
-        returncode, token_val, _ = exec_command(
+        returncode_, token_val, _ = exec_command(
             ['dcos', 'config', 'show', "core.dcos_acs_token"], env)
-        if returncode == 0:
+        if returncode_ == 0:
```

Comment:
Notice that we we're overriding `returncode` [here](https://github.com/dcos/dcos-cli/pull/970/files#diff-3486a2e5d4e0e340bd57f8d6279a2495R200) which gets used [here](https://github.com/dcos/dcos-cli/pull/970/files#diff-3486a2e5d4e0e340bd57f8d6279a2495R214)